apps/api/src/services/chat/core.ts

```diff
@@ -209,8 +212,33 @@ export async function processChatRequest(options: CoreChatOptions) {
 		reasoning_effort,
 		should_think,
 		response_format,
+		stream,
```

Comment:
probably need to check if the provider actually supports streaming / our solution is compatible.